src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/JudicialDecisionNotificationHandler.java

```diff
@@ -224,20 +225,48 @@ private void applicationRequestForInformation(CaseData caseData) {
     }
 
     private void applicationApprovedNotification(CaseData caseData) {
-        if (areRespondentSolicitorsPresent(caseData)) {
+        if (areRespondentSolicitorsPresent(caseData)
+            && useDamageTemplate(caseData)) {
+            caseData.getGeneralAppRespondentSolicitors().forEach(
```

Comment:
Could we have a method call as the same block of code has been used in multiple places if that's applicable?

caseData.getGeneralAppRespondentSolicitors().forEach(
                respondentSolicitor -> sendNotificationForJudicialDecision(
                    caseData, respondentSolicitor.getValue().getEmail(),
                    notificationProperties.getJudgeApproveOrderToStrikeOutDamages()
                ));

Reply:
I will refactor this part of Zikrur code - good point.